files/zh-cn/web/api/broadcastchannel/broadcastchannel/index.md

```diff
@@ -5,25 +5,25 @@ slug: Web/API/BroadcastChannel/BroadcastChannel
 
 {{APIRef("BroadCastChannel API")}}
 
-**`BroadcastChannel()`** 构建函数用于创建一个 {{domxref("BroadcastChannel")}} 对象，并与对应的频道相关联。
+**`BroadcastChannel()`** 构造函数用于创建一个 {{domxref("BroadcastChannel")}} 对象，并与对应的频道相关联。
```

Comment:
I prefer "通道", rather than "频道", as the "channel" is used to communicate with each other - "双方相互**通**信的渠**道**". And this is just the same as the translation in [MessageChannel landing page](https://developer.mozilla.org/zh-CN/docs/Web/API/MessageChannel).

```suggestion
**`BroadcastChannel()`** 构造函数用于创建一个 {{domxref("BroadcastChannel")}} 对象，并与底层的通道相关联。
```